flow/project.py

```diff
@@ -2042,6 +2033,33 @@ def scheduler_jobs(self, scheduler):
         """
         yield from self._expand_bundled_jobs(scheduler.jobs())
 
+    def _get_cached_status(self, cached_status=None):
```

Comment:
The only place where the `cached_status` argument is currently passed is in `get_job_status`. I would be inclined to just have an `if cached_status is None` check in that function rather than optionally pass a `cached_status` argument solely for `get_job_status`.

Reply:
That's a good idea. I was hoping to do that but I didn't recognize how easy it would be to simplify. :smile: